pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -324,70 +379,45 @@ func evictPods(
 }
 
 // sortNodesByUsage sorts nodes based on usage in descending order
-func sortNodesByUsage(nodes []NodeUsageMap) {
+func sortNodesByUsage(nodes []NodeUsage) {
 	sort.Slice(nodes, func(i, j int) bool {
-		var ti, tj api.Percentage
-		for name, value := range nodes[i].usage {
-			if name == v1.ResourceCPU || name == v1.ResourceMemory || name == v1.ResourcePods {
-				ti += value
-			}
-		}
-		for name, value := range nodes[j].usage {
-			if name == v1.ResourceCPU || name == v1.ResourceMemory || name == v1.ResourcePods {
-				tj += value
-			}
-		}
+		ti := nodes[i].usage[v1.ResourceMemory].Value() + nodes[i].usage[v1.ResourceCPU].MilliValue() + nodes[i].usage[v1.ResourcePods].Value()
```

Comment:
do these still make sense to add together now that you're storing absolute values instead of percentages? Like if I have a node with `500 pods, 1 mem, 1 cpu` that will be ranked higher usage than `10 pods, 100 mem, 100 cpu` right? But idk if that's a reasonable comparison

Reply:
I kept the original computation. Though, it might be better to rather carry addition over relative units. E.g. pods/totalpods + cpu/totalcpu + memory/totalmemory. So nodes with higher percentage of consumed resources have higher priority over those with lower percentage. As a feature we might allow to set weight for each resource. E.g. in case memory is more expensive than cpu.